README.md

```diff
@@ -1 +1,35 @@
 # project-alpha
+
+# Installation
+
+Create a new directory to work in, and cd into it:
+```
+mkdir <folder_name>
+cd <folder_name>
+```
+Clone the project:
+```
+git clone https://github.com/asynccat/project-alpha.git .
+```
+Create and activate a virtual environment:
+```
+python3 -m venv venv
+source venv/bin/activate
+```
+cd into backend folder:
+```
+cd backend
+```
+Update pip inside the virtual environment and install project requirements:
+```
+pip install --upgrade pip
+pip install -r requirements.txt
+```
+Run migrations:
```

Comment:
nit: I think it's better to add a note that before running any `./manage.py` command you should run `source venv/bin/activate` to activate virtual env. Otherwise it looks like a part of installation.